scenario-definitions.yaml

```
@@ -0,0 +1,29 @@
+---
+products:
+  opensuse-*-dev-x86_64:
+    distri: opensuse
+    flavor: dev
+    arch: x86_64
+    version: Tumbleweed
+
+machines:
+  64bit:
+    backend: qemu
+    settings:
+      HDDSIZEGB: "20"
+      QEMUCPU: host
+      VIRTIO_CONSOLE: "1"
+      WORKER_CLASS: qemu_x86_64
+
+.common: &common
+  product: opensuse-*-dev-x86_64
+  machine: 64bit
+
+job_templates:
+  boot_to_snapshot:
+    <<: *common
+    settings:
+      BOOT_HDD_IMAGE: "1"
+      DESKTOP: textmode
+      PRODUCTDIR: products/opensuse
+      YAML_SCHEDULE: schedule/boot_to_snapshot.yaml
```

the schedule should be set dinamically:

```suggestion
      YAML_SCHEDULE: schedule/boot_to_snapshot.yaml
```

1. schedule boot_to_desktop and the test module(s) changed if the change is on `tests/`
2. take something from the PR's comment (i.e '#schedule_override: tests/my/very/long,tests/list/of/test.pm,tests/modules/here.pm'
3. Do nothing if there's some pattern in the PR's comment, i.e `Verification Runs`
4. At first, make it explicit, so we don't unnecessarily block things for other test developers (i.e opt in to have your PR tested instead of opt out) until we have a working proof of concept

---

Yes, that's planned for later and needs further discussion. So let's keep it statically defined for now, ok?

---

> Yes, that's planned for later and needs further discussion. So let's keep it statically defined for now, ok?

No :), that would achieve even less than the `os-autoinst` check that we have; let's go with 1 first and then we can distill

---

Is there an open question? The change proposes to schedule *boot_to_desktop*. That's what #1 also says. I would leave further conditionals to a follow-up. The initial version should be simple imho.

---

@foursixnine I would appreciate if you can clarify as you're still "requesting changes". Not sure if you've seen my question.

---

> @foursixnine I would appreciate if you can clarify as you're still "requesting changes". Not sure if you've seen my question.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18182#issuecomment-1868288079

To make it more explicit: There's the "Dismiss review button".